css/store.css

```diff
@@ -0,0 +1,238 @@
+/* store page */
+
+/* main */
+.storemain {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+/* section one */
+.formsection {
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+  padding: 2rem;
+  gap: 30px;
+}
+/* form */
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+/* form title */
+.formsection h1 {
+  color: var(--orange);
+}
+
+/* form classes */
+label {
+  font-size: 12px;
+  font-weight: 400;
+  color: var(--grey-dark);
+}
+
+.fieldset {
+  display: flex;
+  flex-direction: row;
+  align-items: stretch;
+  flex-wrap: wrap;
+  border: none;
+  gap: 30px;
+}
+
+.flex {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  gap: 5px;
+}
+
+/* input style */
+input[type="text"] {
+  height: 30px;
+  padding: 2px;
+}
+
+input:focus {
+  outline-color: var(--orange) !important;
+}
+
+input[type="checkbox"] {
+  color: var(--orange);
+  background-color: var(--orange);
+}
+
+#postcode {
+  width: 100px;
+  text-transform: uppercase;
+}
+/* select & option */
+select {
+  font-style: italic;
+  font-size: 12px;
+  font-weight: 400;
+  color: var(--grey-dark);
+  width: 250px;
+  height: 30px;
+}
+
+select:focus {
+  outline-color: var(--orange);
+}
+
+/* radio-box */
+.radio-box {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  margin-top: 10px;
+}
+
+.radio-box legend {
+  font-size: 12px;
+  font-weight: 400;
+  color: var(--grey-dark);
+}
+
+.radio-box ul {
+  display: flex;
+  gap: 20px;
+  list-style: none;
+}
+
+/* changed radio-box color */
+.radio-box input[type="radio"] {
+  appearance: none;
+  width: 15px;
+  height: 15px;
+  border: 1px solid var(--grey-dark);
+  border-radius: 50%;
+  background-clip: content-box;
+  padding: 2px;
+  display: inline-block;
+  margin-right: 0.1rem;
+  vertical-align: -3px;
+}
+
+.radio-box input[type="radio"]:checked {
+  background-color: var(--orange);
+}
+
+/* check box */
+.check-box {
+  gap: 2px;
+  margin-top: 20px;
+}
+
+.check-box a {
+  color: var(--orange);
+}
+
+/* change checkbox mark */
+
+.check-box input[type="checkbox"] {
```

Comment:
This styles aren't being picked up by the page because the class in the page is `checkbox` not `check-box`